pkg/tasconfigmanager/setup.go

```diff
@@ -36,7 +36,18 @@ func NewTASConfigManager(logger lumber.Logger) core.TASConfigManager {
 	return &tasConfigManager{logger: logger}
 }
 
```

Comment:
Can't we do it the same way as we did for driver. That's why I was saying we should have a folder outside driver. V1 and v2 where we can put multiple pkgs in each file with different implementations

Reply:
This requires some more refactoring , will target this in next PR